src/axom/inlet/Field.hpp

```diff
@@ -23,6 +23,8 @@ namespace axom
 namespace inlet
 {
 
+enum class FieldType {BOOL, INT, DOUBLE, STRING, UNSPECIFIED};
```

Comment:
These are already defined inside of sidre.  I use them in the add functions.  We shouldn't be duplicating definitions.